tests/test_gui.py

```diff
@@ -0,0 +1,64 @@
+from qgis.testing import unittest
+
+from qgis.PyQt.QtTest import QTest
+from qgis.PyQt.QtCore import Qt, QEvent, QPoint
+from qgis.PyQt.QtWidgets import QPushButton
+from qgis.gui import QgsMapCanvas, QgsMapMouseEvent
+from qgis.core import (
+    QgsCoordinateReferenceSystem,
+    QgsRectangle,
+)
+import pytest
+
+from tests.utils.utilities import get_qgis_app
+
+CANVAS: QgsMapCanvas
+QGISAPP, CANVAS, IFACE, PARENT = get_qgis_app()
+
+
+@pytest.mark.filterwarnings("ignore:.*imp module is deprecated.*")
+class TestGui(unittest.TestCase):
+    def test_ORStoolsDialog(self):
+        from ORStools.gui.ORStoolsDialog import ORStoolsDialog
+        from ORStools.utils import maptools
+
+        CRS = QgsCoordinateReferenceSystem.fromEpsgId(3857)
+        CANVAS.setExtent(QgsRectangle(258889, 7430342, 509995, 7661955))
+        CANVAS.setDestinationCrs(CRS)
```

Comment:
```CANVAS = QgsMapCanvas()
        CANVAS.setDestinationCrs(QgsCoordinateReferenceSystem('EPSG:3857'))
        CANVAS.setFrameStyle(0)
        CANVAS.resize(600, 400)
        self.assertEqual(CANVAS.width(), 600)
        self.assertEqual(CANVAS.height(), 400)
```
Otherwise it changes based on the users screen.

Reply:
…what do we do with this? Does it need to be implemented?